SonarQube.TeamBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -80,9 +85,9 @@ protected virtual void Dispose(bool disposing)
         {
             if (!this.disposed && disposing)
             {
-                if (this.Client != null)
+                if (this.client != null)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS1066)